src/main/scala/tile/BaseTile.scala

```diff
@@ -118,6 +118,7 @@ trait HasNonDiplomaticTileParameters {
       Option.when(tileParams.core.useBitManip)(Seq("zbs")) ++
       Option.when(tileParams.core.useCryptoNIST)(Seq("zknd", "zkne", "zknh")) ++
       Option.when(tileParams.core.useCryptoSM)(Seq("zksed", "zksh")) ++
+      Option.when(tileParams.core.mulDiv.nonEmpty && tileParams.core.useMulDivWithoutDiv)(Seq("zmmul")) ++
```

Comment:
1. `zmmul` should be placed after `zihpm` instead of appending to the end

2. Some interesting thing: `rv32imac_zmmul` is a valid ISA string.